doc/sphinx/addendum/type-classes.rst

```diff
@@ -300,7 +318,7 @@ Summary of the commands
 -----------------------
 
 .. cmd:: Class @record_definition
-         Class {? > } @ident_decl {* @binder } {? : @sort } := @constructor
+         Class @ident_decl {* @binder } {? : @sort } := @constructor
```

Comment:
I would not change this line at all because you later explain that it currently does nothing.  Your change would create a puzzle: "where is the `>`?"

Reply:
The one "currently doing nothing" is the one in `record_definition`, this one for singleton class is doing nothing and will keep doing so (it really has no meaning).